images/kubevirt-infra-prow/publish_multiarch_prow_images.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash -x
+
+BAZEL_VERSION=3.0.0
+PROW_COMMIT=master
+repo=kubevirtci
+archs=(arm64 ppc64le)
```

Comment:
Is there a reason why we are not building this image for amd64 also? Is that done somewhere implicitly?

Reply:
OK, so this is just for building the images for other arch's outside of amd64. Understood. But still I wonder whether this could have some value for amd64 envs also.

Reply:
> Is there a reason why we are not building this image for amd64 also? Is that done somewhere implicitly?
Hi, dhiller, prow utility images are already support amd64 and they are maintained by kubernetes community. We tried to push PR and make these images support multiarch, but there is no progress. Following link is discussion about it https://github.com/kubernetes/test-infra/issues/16588

Reply:
Well, I wasn't aware of this. Should be fine then.